ansible/roles/rbenv/tasks/main.yml

```diff
@@ -58,6 +58,18 @@
   become: true
   become_user: deploy
 
+- name: install ruby 2.6.6
+  shell: rbenv install -s 2.6.6
+  environment:
+    PATH: "/home/deploy/.rbenv/bin:/home/deploy/.rbenv/plugins/ruby-builds/bin:{{ ansible_env.PATH }}"
+  become: true
+  become_user: deploy
+
+- name: install bundler for ruby 2.6.6
+  shell: rbenv shell 2.6.6 && ~/.rbenv/shims/gem install bundler -v '1.17.3'
+  become: true
+  become_user: deploy
```

Comment:
@prabhanshuguptagit Is this a safe way to get bundler installed for this newer version of Ruby alongside the ruby 2.5.1 one?

Reply:
Ruby 2.6 changelog indicates that "bundler is a default gem" 🤔  Digging a bit further, [I saw this](https://github.com/ruby/ruby/blob/v2_6_0/NEWS#stdlib-updates-outstanding-ones-only-): 

> Add Bundler to Standard Library. [Feature #12733]

We might not need to explicitly install bundler?

Reply:
Yep, it seems like bundler is installed with the ruby installation, from https://www.ruby-lang.org/en/news/2018/12/25/ruby-2-6-0-released/. Just the ruby installation might work. We can test this on sandbox too to be sure.

Reply:
Just tested locally, and 2.6.6 came with bundler 👍  We can take it for a spin on SBX, agreed.

```
hariharanmohanraj | ~/code/simple/simple-server $ rbenv install 2.6.6
Downloading openssl-1.1.1d.tar.gz...
-> https://dqw8nmjcqpjn7.cloudfront.net/1e3a91bc1f9dfce01af26026f856e064eab4c8ee0a8f457b5ae30b40b8b711f2
Installing openssl-1.1.1d...
Installed openssl-1.1.1d to /Users/hariharanmohanraj/.rbenv/versions/2.6.6

Downloading ruby-2.6.6.tar.bz2...
-> https://cache.ruby-lang.org/pub/ruby/2.6/ruby-2.6.6.tar.bz2
Installing ruby-2.6.6...
ruby-build: using readline from homebrew
Installed ruby-2.6.6 to /Users/hariharanmohanraj/.rbenv/versions/2.6.6

hariharanmohanraj | ~/code/simple/simple-server $ rbenv global 2.6.6
hariharanmohanraj | ~/code/simple/simple-server $ cd
hariharanmohanraj | ~ $ ruby -v
ruby 2.6.6p146 (2020-03-31 revision 67876) [x86_64-darwin19]
hariharanmohanraj | ~ $ bundle help
Bundler commands:
  bundle add GEM VERSION         # Add gem to Gemfile and run bundle install
  bundle binstubs GEM [OPTIONS]  # Install the binstubs of the listed gem
  bundle check [OPTIONS]         # Checks if the dependencies listed in Gemfile are satisfied by currently installed gems
  etc etc etc...
```

Reply:
> Yep, it seems like bundler is installed with the ruby installation now

Awesome, even simpler.